### PR TITLE
chore: remove access of eager load object if not needed

### PR DESCRIPTION
**Pull request by asbiin**
because we can

### CODE REVIEW

File: app/Console/Commands/SetupTest.php [hunk 1]
```diff
@@ -269,7 +269,7 @@ public function populateBirthday()
             $birthdate = $this->faker->dateTimeThisCentury();
 
             app(UpdateBirthdayInformation::class)->execute([
-                'account_id' => $this->contact->account_id,
+                'account_id' => $this->contact->account->id,
```
Comment (djaiss) [inline app/Console/Commands/SetupTest.php]: Nope!

```suggestion
                'account_id' => $this->contact->account_id,
```

File: app/Console/Commands/SetupTest.php [hunk 2]
```diff
@@ -607,7 +607,7 @@ public function populateCalls()
     {
         if (rand(1, 3) == 1) {
             $this->contact->calls()->create([
-                'account_id' => $this->contact->account_id,
+                'account_id' => $this->contact->account->id,
```
Comment (djaiss) [inline app/Console/Commands/SetupTest.php]: All this file is wrong :-D

**Comment by djaiss** [comment]
Just changes in one file and it's good to go!

File: app/Console/Commands/SetupTest.php (hunk 2 above)
Comment (asbiin) [reply app/Console/Commands/SetupTest.php]: That's a bad revert indeed

**Pull request merged**